includes/class-wc-siftscience-options.php

```diff
@@ -13,21 +13,21 @@
 if ( ! class_exists( 'WC_SiftScience_Options' ) ) :
 
 	class WC_SiftScience_Options {
-		public static $mode = 'siftsci_reporting_mode';
-		public static $api_key = 'siftsci_api_key';
-		public static $js_key = 'siftsci_js_key';
-		public static $name_prefix = 'siftsci_name_prefix';
-		public static $is_api_setup = 'siftsci_is_api_setup';
-		public static $send_on_create_enabled = 'siftsci_send_on_create_enabled';
-		public static $send_stats = 'siftsci_send_stats';
-		public static $threshold_good = 'siftsci_threshold_good';
-		public static $threshold_bad = 'siftsci_threshold_bad';
-		public static $log_level_key = 'siftsci_log_level';
-		public static $guid = 'siftsci_guid';
-		public static $stats = 'siftsci_stats';
-		public static $stats_last_sent = 'siftsci_stats_last_sent';
-		public static $min_order_value = 'siftsci_min_order_value';
-		public static $stats_api = 'https://sift.fermiac.staat.us';
+		const API_KEY 					= 'siftsci_api_key';
+		const JS_KEY 					= 'siftsci_js_key';
+		const NAME_PREFIX 				= 'siftsci_name_prefix';
+		const THRESHOLD_GOOD 			= 'siftsci_threshold_good';
+		const THRESHOLD_BAD 			= 'siftsci_threshold_bad';
+		const SEND_ON_CREATE_ENABLED 	= 'siftsci_send_on_create_enabled';
+		const MIN_ORDER_VALUE 			= 'siftsci_min_order_value';
+		public static $mode 			= 'siftsci_reporting_mode';
```

Comment:
Why did you only change half of them?

Reply:
want your approval that you like that write do it for me please

Reply:
Sure, it's a good change. Just two comments:

- The area between the const name and the `=` sign should only have spaces. No tabs.
- Please define all the constants to be public explicitly.

Reply:
constants are public 
I'll write it ~OK